commons/src/main/kotlin/com/simplemobiletools/commons/extensions/Activity.kt

```diff
@@ -727,36 +726,45 @@ fun BaseSimpleActivity.deleteFilesBg(files: List<FileDirItem>, allowDeleteFolder
 
             if (canManageMedia()) {
                 val fileUris = getFileUrisFromFileDirItems(files).second
-                deleteSDK30Uris(fileUris) { success ->
-                    runOnUiThread {
-                        callback?.invoke(success)
+                if(fileUris.size == files.size){
```

Comment:
seems like a quite tricky check, have you tested if it works properly with SD cards too etc? I thought that you will just add a recycle bin check itself. And please format the code style with ctrl + alt + l

Reply:
will do some tests and see

Reply:
Fixed in #1427